fedimint-core/src/module/mod.rs

```diff
@@ -924,7 +935,7 @@ impl<T: Encodable + Decodable + 'static> SerdeModuleEncoding<T> {
             ModuleInstanceId::consensus_decode(&mut reader, &ModuleDecoderRegistry::default())?;
 
         let _total_len =
-            u32::consensus_decode(&mut reader, &ModuleDecoderRegistry::default())? as usize;
+            u64::consensus_decode(&mut reader, &ModuleDecoderRegistry::default())? as usize;
```

Comment:
I assume switching from `u32` to `u64` is in anticipation of #2752 or is there another reason for it?

Reply:
This is to unify it with `Vec<u8>` which already was using `u64`.